app/models/embeddable_host.rb

```diff
@@ -53,6 +54,13 @@ def self.url_allowed?(url)
 
   private
 
+  def reset_embedding_settings
+    unless EmbeddableHost.any?
```

Comment:
Can we use the positive `empty?` or `blank?` instead of `not .any?`?

```suggestion
    if EmbeddableHost.empty?
```